platform/freedesktop/detect.py

```diff
@@ -8,7 +8,7 @@ def is_active():
 
 
 def get_name():
-    return "X11"
+    return "Linux/BSD"
```

Comment:
Maybe I'm wrong, but I get the feeling this might be a breaking change, despite being innocent enough.